content/content-strategy/removing-content/archive-record-content/archiving-comm02.md

```diff
@@ -3,7 +3,6 @@ layout: text/textblock
 section: Archiving for Commonwealth records
 ---
 ## [2]If your website is not stored with the National Library
-
-If the site you’re removing is not archived with the AGWA, email them and make sure it's captured in their next website harvest: [webarchive@nla.gov.au](mailto:webarchive@nla.gov.au)
+If the site you’re removing is not archived with the AGWA, email them and make sure it's captured in their next website harvest: [recordkeeping@naa.gov.au](mailto:recordkeeping@naa.gov.au)
```

Comment:
This commit is wrong, should be the `master` version (`NLA`).

Reply:
@joolswood would you mind editing this then here? So we can approve all?